man/rmd/bake_dependent_roles.Rmd

```diff
@@ -0,0 +1,55 @@
+## Effects of non-standard roles
+
+
+Recipes steps can keep column(s) of your data set that should not be treated as outcomes or predictors. For example, a unique identifier column or some other ancillary data that can be used to troubleshoot issues during model development. 
+
+As an example, you might have a date column that is transformed into features using [step_date()] and.or [step_holiday()]. 
+
+```{r, results = "hide", message = FALSE, warning = FALSE}
+  library(tidymodels)
```

Comment:
It looks like all of the code in your code chunks are indented, was that on purpose?

Reply:
it is!